src/solver/ts-generator/xcast/xcast.cpp

```diff
@@ -43,12 +43,13 @@ using namespace Yuni;
 
 namespace Antares
 {
-namespace Solver
-{
 namespace TSGenerator
 {
 namespace XCast
 {
+
+const unsigned int nbHoursADay = 24;
```

Comment:
Maybe define in in constants.h with HOURS_PER_YEAR ?
It's fine declaring it locally so up to you

Reply:
Done